tests/core/test_processor.py

```diff
@@ -168,18 +171,13 @@
     # retrieve the updated tracker
     t = default_processor.tracker_store.retrieve(sender_id)
 
-    assert t.events[-4] == UserUttered(None)
-    assert t.events[-3] == ActionExecuted("utter_greet")
-    assert t.events[-2] == BotUttered(
-        "hey there None!",
-        {
-            "elements": None,
-            "buttons": None,
-            "quick_replies": None,
-            "attachment": None,
-            "image": None,
-            "custom": None,
-        },
+    assert t.events[-5] == UserUttered("test")
+    assert t.events[-4] == ActionExecuted("action_schedule_reminder")
+    assert isinstance(
+        t.events[-3], ReminderScheduled
+    )  # ToDo: Should this really be here?
```

Comment:
What's up with this Todo? Can we resolve it?

Reply:
Ah, yes. That was just a note. It should be ok this way.